src/ansys/grantami/recordlists/_models.py

```diff
@@ -3,7 +3,8 @@
 from enum import Enum
 from typing import List, Optional, Union
 
-from ansys.grantami.serverapi_openapi import models  # type: ignore
+from ansys.grantami.serverapi_openapi import models  # type: ignore[import]
+from ansys.openapi.common import Unset  # type: ignore[import]
```

Comment:
Both should be typed now? Perhaps for another PR

Reply:
Indeed. It will require running mypy from the venv, which means tweaks to pre-commit config and CI

Reply:
#188